inst/rmarkdown/templates/springer/skeleton/skeleton.Rmd

```diff
@@ -1,63 +1,380 @@
+
+\begin{theorem}[Theorem subhead]\label{thm1}
+Example theorem text. Example theorem text. Example theorem text. Example theorem text. Example theorem text.
+Example theorem text. Example theorem text. Example theorem text. Example theorem text. Example theorem text.
+Example theorem text.
+\end{theorem}
+
+
+Other environments are proposition, example, remark, definition, proof and quote
+
+
+
+# Methods {#sec11}
+
+Topical subheadings are allowed. Authors must ensure that their Methods section includes adequate experimental and characterization data necessary for others in the field to reproduce their work. Authors are encouraged to include RIIDs where appropriate.
+
+**Ethical approval declarations** (only required where applicable) Any article reporting experiment/s carried out on (i)~live vertebrate (or higher invertebrates), (ii)~humans or (iii)~human samples must include an unambiguous statement within the methods section that meets the following requirements:
+
+1. Approval: a statement which confirms that all experimental protocols were approved by a named institutional and/or licensing committee. Please identify the approving body in the methods section
+
+2. Accordance: a statement explicitly saying that the methods were carried out in accordance with the relevant guidelines and regulations
+
+3. Informed consent (for experiments involving humans or human tissue samples): include a statement confirming that informed consent was obtained from all participants and/or their legal guardian/s
+
+
+If your manuscript includes potentially identifying patient/participant information, or if it describes human transplantation research, or if it reports results of a clinical trial then  additional information will be required. Please visit (<https://www.nature.com/nature-research/editorial-policies>) for Nature Portfolio journals, (<https://www.springer.com/gp/authors-editors/journal-author/journal-author-helpdesk/publishing-ethics/14214>) for Springer Nature journals, or (<https://www.biomedcentral.com/getpublished/editorial-policies\#ethics+and+consent>) for BMC.
+
+# Discussion {#sec12}
+
+Discussions should be brief and focused. In some disciplines use of Discussion or 'Conclusion' is interchangeable. It is not mandatory to use both. Some journals prefer a section 'Results and Discussion' followed by a section 'Conclusion'. Please refer to Journal-level guidance for any specific requirements.
+
+# Conclusion {#sec13}
+
+Conclusions may be used to restate your hypothesis or research question, restate your major findings, explain the relevance and the added value of your work, highlight any limitations of your study, describe future directions for research and recommendations.
+
+In some disciplines use of Discussion or 'Conclusion' is interchangeable. It is not mandatory to use both. Please refer to Journal-level guidance for any specific requirements.
+
+\backmatter
+
+\bmhead{Supplementary information}
```

Comment:
I don't think so. [Someone is having a similar problem](https://stackoverflow.com/questions/75321227/adding-classes-to-headers-with-a-pandoc-lua-filter) trying to adapt this template to quarto and their solution is creating a custom lua filtre, which might be overkill here (and I don't know how to do it :P).

Reply:
Yeah we could also do it in rticles with Lua filter but I agree that this could be too much. 

For Quarto, there is already a `latex-environment` extension that handles command that could be useful for this

Let's leave it